packages/apps-config/src/api/spec/dataavail.ts

```diff
@@ -36,6 +36,22 @@ const definitions: OverrideBundleDefinition = {
         ],
         type: 'Vec<u8>'
       },
+      queryAppData: {
+        description: 'Fetches app data rows for the given app',
+        params: [
+          {
+            name: 'app_id',
+            type: 'AppId'
+          },
+          {
+            name: 'at',
+            type: 'Hash',
+            isOptional: true
+          },
+        ],
+        type: 'Vec<Vec<u8>>'
```

Comment:
The output looks different from the RPC here: [kate-rpc/src/lib.rs](https://github.com/maticnetwork/avail/blob/641a758e317d78bb021faa93416d65d3c150ad1c/rpc/kate-rpc/src/lib.rs#L133)


```suggestion
        type: 'Vec<Option<Vec<u8>>>'
```